index.js

```diff
@@ -15,16 +29,16 @@ const aliased = [
   'utils',
 ];
 
-function isInternalModule({ moduleName }) {
-  return moduleName.startsWith('@qsi/');
+function isScopedModule({ moduleName }) {
```

Comment:
all scoped modules though? isn't there a bunch of public stuff scoped under `@exponent` or whatever?